src/components/general/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import * as styles from "./styles.less";
+
+type TabsProps = {
+    onTabClick: (tabKey: string) => void,
```

Comment:
Should this be onChange instead of onTabClick? Makes more sense if we're to support keyboard navigation etc.